lib/myfinance/resources/receivable_account.rb

```diff
@@ -0,0 +1,24 @@
+module Myfinance
+  module Resources
+    #
+    # A wrapper to Myfinance receivable accounts API
+    #
+    # [API]
+    #   Documentation: https://app.myfinance.com.br/docs/api/receivable_accounts
+    #
+    class ReceivableAccount < FinancialAccount
+
+      private
+
+      def endpoints
+        {
+          create: '/entities/:entity_id/receivable_accounts'
+        }
+      end
+
+      def resource_key
+        'receivable_account'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.